tools/state-viewer/src/commands.rs

```diff
@@ -652,14 +652,12 @@ pub(crate) fn replay_chain(
         near_config.client_config.save_trie_changes,
     );
     let new_store = create_test_store();
-    let epoch_manager =
-        EpochManager::new_arc_handle(new_store.clone(), &near_config.genesis.config);
-    let runtime = NightshadeRuntime::from_config(home_dir, new_store, &near_config, epoch_manager);
```

Comment:
Interesting, we no longer need a nightshade runtime to "replay" blocks. I guess that makes sense and indicates that the method really better belongs to the epoch manager.